### PR TITLE
[ZH| The Team

### PR DESCRIPTION
**Pull request by MinecraftDeadBush**

### CODE REVIEW

**Comment by sr229** [comment]
Based on the contents of the article, I believe @kj415j45 is the best reviewer for this.

File: wiki/People/The_Team/zh.md [hunk 1]
```diff
@@ -1,56 +1,87 @@
...
-| [nanaya](https://osu.ppy.sh/users/2387883) | 网页成绩 |
-| [RBRat3](https://osu.ppy.sh/users/307202) | osu! 画师, 制作很酷的素材 |
-| [smoogipooo](https://osu.ppy.sh/users/1040328) | osu!mania 开发, 发掘 Bug |
-| [Tom94](https://osu.ppy.sh/users/1857058) | pp 系统开发 |
-| [yelle](https://osu.ppy.sh/users/4916903) | 举办美术竞赛, osu!store 管理 |
-| [ztrot](https://osu.ppy.sh/users/6347) | osu! 学院教授, 音频/文字设计 |
-| [Zallius](https://osu.ppy.sh/users/55) | 濒危物种 |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | Moderation<!--NT--> 和社区的顾问 |
+| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | 项目/社区的维护和[精选艺术家](https://osu.ppy.sh/beatmaps/artists)的推广 |
+| ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765) | 外观设计设定 lead<!--NT--> 和 resident Creative Cloud watcher<!--NT--> |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765) | 首席设计师、Creative Cloud 常任观察员 |
```

File: wiki/People/The_Team/zh.md [hunk 2]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) | 摸图和作图圈的顾问 |
+| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | 世界杯的组织和管理者 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | 世界杯的组织者和管理者 |
```

File: wiki/People/The_Team/zh.md [hunk 3]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | 摸图和作图圈的顾问 |
+| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | 摸图和作图圈的顾问 |
+| ![][flag_JP] [nanaya](https://osu.ppy.sh/users/2387883) | osu!web 开发者 |
+| ![][flag_PH] [Nathanael](https://osu.ppy.sh/users/2295078) | 基本秩序维护 & moderation<!--NT-->顾问 |
+| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | osu!web 开发者 & [官方 osu! 特色艺术家](https://osu.ppy.sh/beatmaps/artists/1) |
+| ![][flag_JP] [notbakaneko](https://osu.ppy.sh/users/10751776) | osu!web 开发者 |
+| ![][flag_US] [Nyquill](https://osu.ppy.sh/users/682935) | osu! 开发者, osu! 开发者, Mappers' Guild 组织者 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_US] [Nyquill](https://osu.ppy.sh/users/682935) | osu! 开发者、Mappers' Guild 组织者 |
```

File: wiki/People/The_Team/zh.md [hunk 4]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) | 摸图和作图圈的顾问 |
+| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | 摸图和作图圈的顾问 & Mappers' Guild 组织者 & [osu!academy](/wiki/osu!academy) 教授 |
+| ![][flag_MX] [Repflez](https://osu.ppy.sh/users/201392) | 基本秩序维护 & wiki 维护 |
+| ![][flag_FR] [Shiro](https://osu.ppy.sh/users/113005) | 基本秩序维护 & 特色艺术家助手 |
+| ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328) | osu! 开发者, osu!mania 爱好者, 修bug |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328) | osu! 开发者、osu!mania 爱好者、bug 杀手 |
```

File: wiki/People/The_Team/zh.md [hunk 5]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Project Loved 队长 & 社区顾问 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Project Loved 领导者 & 社区顾问 |
```

File: wiki/People/The_Team/zh.md [hunk 6]
```diff
@@ -1,56 +1,87 @@
...
-| [Intermezzo](https://osu.ppy.sh/users/136842) | osu! dev 团队, osz2/p2p 后台 |
-| Jim ([Brand New Games](http://www.bravegamer.com/)) |  原创网页设计 |
-| [LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](http://luigihann.deviantart.com/)) | 杰出的皮肤设计, 为默认皮肤和图标的设计做出贡献 |
-| [mm201](https://osu.ppy.sh/users/30655) | osu! dev 团队, mm 滑条 |
-| [nuudles](https://osu.ppy.sh/users/21312) | osu! iPhone dev |
-| Phil | 论坛设计工作 |
-| [Sarumaru](https://osu.ppy.sh/users/9427)  ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | 原创的 pippi 概念和美术设计 |
-| [Sinistro](https://osu.ppy.sh/users/5530) | 讨论记录, FAQ 贡献者 |
-| [woc2006](https://osu.ppy.sh/users/1105845) | osu! dev 团队, osu!mania 模式创始人 |
+| ![][flag_US] [awp](https://osu.ppy.sh/users/2650) | 基本秩序维护和帮助, 游戏/网页上 "Chibi" 的美术设计 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_US] [awp](https://osu.ppy.sh/users/2650) | 基本秩序维护和帮助, 游戏/网页上“Chibi”的美术设计，[Barrel Roll Weekly](http://brw.twinkfish.com/) |
```

File: wiki/People/The_Team/zh.md [hunk 7]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | 音频设计 & [官方 osu! 特色艺术家](https://osu.ppy.sh/beatmaps/artists/2) |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | 世界杯的组织和管理者 |
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | 基本秩序维护, 社区管理 |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | osu! dev 团队, 游戏内聊天的一体化, 网页维护 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | osu! 开发团队, 游戏内聊天的一体化, 网页维护，[博客](http://blog.echo.sh/) |
```

File: wiki/People/The_Team/zh.md [hunk 8]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour 房主 & 世界杯流媒体和比赛组织者 |
+| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | osu! 开发者团队, osz2/p2p 后台 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | osu! 开发团队, osz2/p2p 后端 |
```

File: wiki/People/The_Team/zh.md [hunk 9]
```diff
@@ -1,56 +1,87 @@
...
-| Phil | 论坛设计工作 |
-| [Sarumaru](https://osu.ppy.sh/users/9427)  ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | 原创的 pippi 概念和美术设计 |
-| [Sinistro](https://osu.ppy.sh/users/5530) | 讨论记录, FAQ 贡献者 |
-| [woc2006](https://osu.ppy.sh/users/1105845) | osu! dev 团队, osu!mania 模式创始人 |
+| ![][flag_US] [awp](https://osu.ppy.sh/users/2650) | 基本秩序维护和帮助, 游戏/网页上 "Chibi" 的美术设计 |
+| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | 音频设计 & [官方 osu! 特色艺术家](https://osu.ppy.sh/beatmaps/artists/2) |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | 世界杯的组织和管理者 |
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | 基本秩序维护, 社区管理 |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | osu! dev 团队, 游戏内聊天的一体化, 网页维护 |
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour 房主 & 世界杯流媒体和比赛组织者 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour 组织者 & 世界杯直播员、组织者 |
```

File: wiki/People/The_Team/zh.md [hunk 10]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | osu! 开发者团队, osz2/p2p 后台 |
+| ![][flag_US] Jim | 原创网页设计, [Brand New Games](http://www.bravegamer.com/) |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_US] Jim | 初代网页设计, [Brand New Games](http://www.bravegamer.com/) |
```

File: wiki/People/The_Team/zh.md [hunk 11]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | 世界杯组织和管理者 & QAT leadership<!--NT--> & 社区管理 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | 世界杯组织者、管理者 & QAT 领导者 & 社区管理 |
```

File: wiki/People/The_Team/zh.md [hunk 12]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) | 杰出的皮肤设计, 为默认皮肤和图标的设计做出贡献, [DeviantArt](https://luigihann.deviantart.com/) |
+| ![][flag_US] [mm201](https://osu.ppy.sh/users/30655) | osu! dev 团队, mm 滑条 |
+| ![][flag_US] [nuudles](https://osu.ppy.sh/users/21312) | osu! iPhone 开发者 |
+| ![][flag_US] Phil | 论坛设计工作 |
+| ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202) | osu! 画师, 制作很酷的素材 |
+| ![][flag_US] [Sarumaru](https://osu.ppy.sh/users/9427) | 原创的 pippi 概念和美术设计, [DeviantArt](https://sarumaru.deviantart.com/) |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_US] [Sarumaru](https://osu.ppy.sh/users/9427) | 初代 pippi 概念与美术设计, [DeviantArt](https://sarumaru.deviantart.com/) |
```

File: wiki/People/The_Team/zh.md [hunk 13]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_GR] [Sinistro](https://osu.ppy.sh/users/5530) | 早期社区管理与leadership<!--NT--> & 讨论记录 & FAQ 贡献者 |
+| ![][flag_DE] [Tom94](https://osu.ppy.sh/users/1857058) | pp 系统开发者 |
+| ![][flag_CN] [woc2006](https://osu.ppy.sh/users/1105845) | osu! 开发者团队, osu!mania 模式创始人 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_CN] [woc2006](https://osu.ppy.sh/users/1105845) | osu! 开发团队, osu!mania 开发者 |
```
创始人[来源请求]

File: wiki/People/The_Team/zh.md [hunk 14]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_JP] [yelle](https://osu.ppy.sh/users/4916903) | 举办美术竞赛 & [osu!store](https://osu.ppy.sh/store/listing) 管理 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| ![][flag_JP] [yelle](https://osu.ppy.sh/users/4916903) | [osu!store](https://osu.ppy.sh/store/listing) 管理 |
```

File: wiki/People/The_Team/zh.md [hunk 15]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_US] [ztrot](https://osu.ppy.sh/users/6347) | [osu!academy](/wiki/osu!academy) 教授 & 媒体<!--NC:Media翻译为媒体是否准确-->/人物<!--NC:Character翻译为人物是否准确--> 设计 |
 
 ## 用户组
 
 | 名 | 描述 |
 | ---- | ----------- |
-| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | 管理论坛和聊天室 |
-| [Beatmap Nomination Group](/wiki/People/Beatmap_Nomination_Group) | 使你的图能够上架的普通用户 |
+| [Beatmap Nominators](/wiki/People/Beatmap_Nominators) | 使你的图能够上架的普通用户 |
 | [Developers](https://osu.ppy.sh/groups/11) | 通过添加新功能和修复 Bugs 使游戏变得有趣 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| [Developers](https://osu.ppy.sh/groups/11) | 通过添加新功能和修复 bugs 使游戏变得有趣 |
```

File: wiki/People/The_Team/zh.md [hunk 16]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_JP] [yelle](https://osu.ppy.sh/users/4916903) | 举办美术竞赛 & [osu!store](https://osu.ppy.sh/store/listing) 管理 |
+| ![][flag_US] [ztrot](https://osu.ppy.sh/users/6347) | [osu!academy](/wiki/osu!academy) 教授 & 媒体<!--NC:Media翻译为媒体是否准确-->/人物<!--NC:Character翻译为人物是否准确--> 设计 |
 
 ## 用户组
 
 | 名 | 描述 |
 | ---- | ----------- |
-| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | 管理论坛和聊天室 |
-| [Beatmap Nomination Group](/wiki/People/Beatmap_Nomination_Group) | 使你的图能够上架的普通用户 |
+| [Beatmap Nominators](/wiki/People/Beatmap_Nominators) | 使你的图能够上架的普通用户 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| [Beatmap Nominators](/wiki/People/Beatmap_Nominators) | 使你的图能够上架的玩家 |
```

File: wiki/People/The_Team/zh.md [hunk 17]
```diff
@@ -1,56 +1,87 @@
...
 | [Developers](https://osu.ppy.sh/groups/11) | 通过添加新功能和修复 Bugs 使游戏变得有趣 |
-| [osu! alumni](/wiki/People/osu!_Alumni) | 曾经为社区做出了杰出贡献 |
-| [Support Team Redux](/wiki/People/Support_Team) | 处理帮助/新功能请求的助手 |
-| [osu!news](https://osu.ppy.sh/groups/25) | 从2013年开始给你带来高质量的阅读体验 |
-| [Tournament Staff](https://osu.ppy.sh/groups/26) | 为所有模式举办官方的 osu! 联赛 |
+| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | 管理论坛和聊天室 |
+| [osu! Alumni](/wiki/People/osu!_Alumni) | 曾经为社区做出了杰出贡献 |
+| [Project Loved Team](/wiki/People/Project_Loved_Team) | 辨认社区最喜欢的谱面 |
 | [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) | 审核将要上架谱面的质量 |
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
| [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) | 通过审核将要上架谱面的质量，提升你的 osu! 体验 |
```

File: wiki/People/The_Team/zh.md [hunk 18]
```diff
@@ -1,56 +1,87 @@
...
+以下列表中的用户<!--NC:翻译为用户是否合适-->是 osu!team 的核心成员，并且负责让这个游戏变得更有趣的成员<!--NC:需要确认原句"and are chiefly responsible for helping make the game awesome for everyone."的翻译是否合理--> 。
+
 ## 游戏创建者
 
 | 名 | 描述 |
 | ---- | ----------- |
-| [peppy](https://osu.ppy.sh/users/2) | 项目领头人 |
+|  ![][flag_AU][peppy](https://osu.ppy.sh/users/2) | 项目领头人 |
 
 ## 活跃用户
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
## 活跃
```

File: wiki/People/The_Team/zh.md [hunk 19]
```diff
@@ -1,56 +1,87 @@
...
+| ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) | 摸图和作图圈的顾问 |
+| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | 摸图和作图圈的顾问 & Mappers' Guild 组织者 & [osu!academy](/wiki/osu!academy) 教授 |
+| ![][flag_MX] [Repflez](https://osu.ppy.sh/users/201392) | 基本秩序维护 & wiki 维护 |
+| ![][flag_FR] [Shiro](https://osu.ppy.sh/users/113005) | 基本秩序维护 & 特色艺术家助手 |
+| ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328) | osu! 开发者, osu!mania 爱好者, 修bug |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Project Loved 队长 & 社区顾问 |
+| ![][flag_ES] [Trosk-](https://osu.ppy.sh/users/3469385) | 基本秩序维护 & 赛事助手 |
+| ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55) | 濒危物种 |
 
 ## 不活跃用户
```
Comment (kj415j45) [inline wiki/People/The_Team/zh.md]:
```suggestion
## 不活跃
```

**Comment by MinecraftDeadBush** [comment]
是不是还要把"&"改成"，"

**Comment by kj415j45** [comment]
没有必要，可以留着

**Pull request closed without merging**